cristin-person/src/test/java/no/unit/nva/cristin/person/create/CreateCristinPersonHandlerTest.java

```diff
@@ -67,6 +73,9 @@ public class CreateCristinPersonHandlerTest {
     private static final String DUMMY_FIRST_NAME = randomString();
     private static final String DUMMY_LAST_NAME = randomString();
     private static final String DUMMY_CRISTIN_ID = "123456";
+    private static final String ANOTHER_ORGANIZATION = "https://api.dev.nva.aws.unit.no/cristin/organization/20202.0.0"
```

Comment:
Here, you're using real examples, but you could test with https://example.org/20000.x.y.z, since we parse the number uniformly as the last path element